packages/components/pie-radio/src/radio.scss

```diff
@@ -50,6 +50,8 @@
 
     @include radio-interactive-state('dt-color-interactive-brand');
 
+
```

Comment:
nit: additional / unnecessary line breaks?

Reply:
strange the linter didnt spot these! I will remove in a subsequent PR if you don't mind :)